crates/bevy_ecs/src/query/filter.rs

```diff
@@ -574,6 +574,29 @@ macro_rules! impl_flag_filter {
 
 impl_flag_filter!(
     /// Filter that retrieves components of type `T` that have been added since the start of the frame
+    ///
+    /// This filter is useful as a performance optimization as it means that a system can be made to run only when
```

Comment:
This isn't _quite_ true. The system will still run, but the query will only return changed items. Its a small distinction, but I think we should be crisp here.

Reply:
Indeed! I had to think about this one, but yep, system execution != query result. Thanks for pointing this out.